src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFAMainPage.java

```diff
@@ -34,4 +34,11 @@ public String getAccountHomePage() {
         return CommonActions.getElement(this.homeContact).getText();
     }
 
+    /**
+     * Wait until the page loads.
+     */
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)